gitbootcamp.rst

```diff
@@ -551,3 +551,42 @@ To edit an open pull request that targets ``main``:
       git push git@github.com:<contributor>/cpython <pr_number>:<branch_name>
 
 5. Optionally, :ref:`delete the PR branch <deleting_branches>`.
+
+
+GitHub CLI
+----------
+
+WIP: short description.
+
```

Comment:
It should include the link to the installation instructions.

https://github.com/cli/cli#installation

Reply:
@Mariatta Added.